docs/docs/30-administration/22-backends/20-local.md

```diff
@@ -92,6 +88,17 @@ steps:
       [...]
 ```
 
+### Plugins as Executable Binaries
+
+```yaml
+steps:
+  build:
+    image: /usr/bin/tree
```

Comment:
As this is not an image should we consider adding a different property maybe to stay clean?

Reply:
I would say stay clean as it is now ...

the local backend is not the main focus of woodpecker ... it's just to cover all edgecases